### DISCUSSION TITLE
It's related to ping time out.

### DISCUSSION THREAD

**Discussion by R3ALW1NNER** [inspircd/inspircd#1952] [Support]
Is ping freq the only attribute related to ping timeout in Inspircd in <connect>?

When pingfreq is changed to 30s, PONG is sent properly, but the connection is continuously disconnected to [Ping timeout: 30 seconds].

Additionally, the version is Inspircd-3.11.0

Fri Dec 10 2021 14:49:45 USERINPUT: C[888AAAAAK] I PING HISS
Fri Dec 10 2021 14:49:45 USEROUTPUT: C[888AAAAAK] O @time=2021-12-10T05:49:45.557Z :irc.localhost PONG irc.localhost :HISS
Fri Dec 10 2021 14:50:15 USERS: QuitUser: 888AAAAAK=system 'Ping timeout: 30 seconds'
Fri Dec 10 2021 14:50:15 USEROUTPUT: C[888AAAAAK] O @time=2021-12-10T05:50:15.564Z ERROR :Closing link: (system@127.0.0.1) [Ping timeout: 30 seconds]
Fri Dec 10 2021 14:50:15 SOCKET: Remove file descriptor: 9
Fri Dec 10 2021 14:50:15 QUIT: Client exiting: system!system@127.0.0.1 (127.0.0.1) [Ping timeout: 30 seconds]

**Comment by sadiepowell** [comment]
That looks like your client is sending a PING to the server and getting a PONG. Your client also need to respond to PING requests sent by the server.
